GCRCatalogs/alphaq.py

```diff
@@ -65,7 +66,9 @@ def _subclass_init(self, filename, **kwargs):
             'size_bulge_true':    'morphology/spheroidHalfLightRadius',
             'disk_sersic_index':  'morphology/diskSersicIndex',
             'bulge_sersic_index': 'morphology/spheroidSersicIndex',
-            'ellipticity_1':      'morphology/totalEllipticity1',
+            'position_angle':     (lambda pos_angle: pos_angle*(180.0/np.pi)**2,'morphology/positionAngle'),
+            'ellipticity_1':      (lambda ellip2, pos_angle : ellip2/np.tan(2*pos_angle*(180.0/np.pi)) , 'morphology/totalEllipticity2','morphology/positionAngle'),
```

Comment:
- use `np.rad2deg` instead of `*(180.0/np.pi)`
- Please add one space after commas, and remove spaces before commas
- Please remove the space before colon